### DISCUSSION TITLE
Maps hitsounded in stable should keep the classic default skin hitsounds

### DISCUSSION THREAD

**Discussion by ProfessionalMD5** [ppy/osu#13479] [Ideas] [answered]
I understand the need of "modernizing" the default skin hitsounds in Lazer, and while I think most of them are way different than the melody of songs (the classic ones are more accurate) the maps hitsounded using previous classic hitsounds should at least keep using the classic hitsounds in fairness to the mapper.

My main issue is that the new hitsounds are WAY too different than the classic ones, and the expected hitsound behaviour that the mapper intended to have while hitsound with the classic default skin will be totally lost.

I take this example from my own ranked maps, that looks terribly hitsounded when the new default skin is used, and it feels really sad to know that this is what players will hear when they play my maps on Lazer.

It's also important to notice that even maps with custom hitsounds will rely on the default classic hitsounds sometimes, so even those can have unexpected behaviour while played on Lazer.

I understand that you can still use the classic skin on Lazer, but that's not viable as most players won't be using it, and even now that old classic elements are used in older skins, I am not sure how this will adapt if players are using modern skins.

TL;DR -> my proposal basically is to add a mark for ALL maps ranked and mapped on stable to use the classic hitsounds from default skin (when not custom hitsounded) instead of the modern default ones added on Lazer.

**Comment by peppy** [comment, marked as answer]
The hit sounds will be changed, current samples are placeholder.
